lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -184,7 +193,7 @@ class LighthouseReportViewer {
       }
 
       // Only clear query string if current report isn't from a gist or PSI.
-      if (!this._reportIsFromGist && !this._reportIsFromPSI) {
+      if (!this._reportIsFromGist && !this._reportIsFromPSI && !this._reportIsFromJSON) {
```

Comment:
also need to update line 208 `finally` reset

Reply:
oh good catch, done at https://github.com/Kikobeats/lighthouse/commit/5413d9c8f072e67d400410f1e51b016fac037b68 🙂